pandas/plotting/_core.py

```diff
@@ -1980,52 +1980,148 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
 
 
 _shared_docs['boxplot'] = """
-    Make a box plot from DataFrame column optionally grouped by some columns or
-    other inputs
+    Make a box plot from DataFrame columns.
+
+    Make a box-and-whisker plot from DataFrame columns optionally grouped
+    by some other columns. A box plot is a method for graphically depicting
+    groups of numerical data through their quartiles.
+    The box extends from the Q1 to Q3 quartile values of the data,
+    with a line at the median (Q2).The whiskers extend from the edges
+    of box to show the range of the data. The position of the whiskers
+    is set by default to 1.5*IQR (IQR = Q3 - Q1) from the edges of the box.
+    Outlier points are those past the end of the whiskers.
+
+    For further details see
+    Wikipedia's entry for `boxplot <https://en.wikipedia.org/wiki/Box_plot>`_.
 
     Parameters
     ----------
-    data : the pandas object holding the data
-    column : column name or list of names, or vector
-        Can be any valid input to groupby
-    by : string or sequence
-        Column in the DataFrame to group by
-    ax : Matplotlib axes object, optional
-    fontsize : int or string
-    rot : label rotation angle
+    column : str or list of str, optional
+        Column name or list of names, or vector.
+        Can be any valid input to groupby.
+    by : str or array-like
+        Column in the DataFrame to groupby.
+    ax : object of class matplotlib.axes.Axes, default `None`
```

Comment:
keeping it as optional is fine. In the explanation below, I would add that if not passed, a new instance is created.

Reply:
Ok, thought the same. Will add it.